quodlibet/browsers/__init__.py

```diff
@@ -1,23 +1,23 @@
 # Copyright 2004-2005 Joe Wreschnig, Michael Urman, Iñigo Serna
 #           2012 Christoph Reiter
-#           2016-21 Nick Boultbee
+#           2016-23 Nick Boultbee
```

Comment:
This copyright update is not consistently applied throughout all of the changed files.